consensus/src/dag/dag_driver.rs

```diff
@@ -139,14 +139,31 @@ impl DagDriver {
 
     pub async fn enter_new_round(&mut self, new_round: Round, strong_links: Vec<NodeCertificate>) {
         debug!("entering new round {}", new_round);
-        // TODO: support pulling payload
+        let payload_filter = {
+            let dag_reader = self.dag.read();
+            let highest_commit_round = dag_reader.highest_committed_anchor_round();
```

Comment:
we should go as far as highest_commit_round - DAG_WINDOW and select uncommitted payloads as well?

Reply:
good point, maybe we should just reach lowest round for now (with gc it should be approximate the highest_commit - window?)

Reply:
sounds good.

Reply:
Actually, it is a trade-off. Maybe we do want to help nodes that are outside of the anchor's causal history?